examples/js/geometries/TeapotGeometry.js

```diff
@@ -70,7 +70,7 @@
 			super(); // number of segments per patch
 
 			segments = Math.max( 2, Math.floor( segments ) ); // Jim Blinn scaled the teapot down in size by about 1.3 for
-			// some rendering tests. He liked the new proportions that he kept
+			// some rendering tests. They liked the new proportions that they kept
```

Comment:
Maybe you should ask Jim Blinn if he prefers to be referred to as "they".

Reply:
Haha, that made my day 😁

Reply:
Yeah, lets revert this one 😁